File: AspNetCore/TestResource.Domain/Model/BaseReturn.cs
namespace TestResource.Domain.Model
{
    public class BaseReturn<T>
    {
        public BaseReturn()
        {
            Success = true;
            Description = null;
            Data = default(T);
        }

        public bool Success { get; set; }

        public string Description { get; set; }

        public T Data { get; set; }
    }
}

File: Spring/src/main/java/com/test/model/BaseReturn.java
package com.test.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Builder
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class BaseReturn<T> {
    public BaseReturn() {
        Success = true;
        Description = null;
        Data = null;
    }

    public Boolean Success;

    public String Description;

    public T Data;
}